seed/data/notificationType/notificationTypes.js

```diff
@@ -1,17 +1,14 @@
-export default [
+module.exports = [
   {
     code: 'SET-APPOINTMENT',
     name: 'Type #1',
     description: 'Test notification type',
     template_id: 1,
```

Comment:
We need to figure out a way to associate the correct template ids with the notification type. I'm thinking we need to manage the notification types together with the templates. So we either move these into the template management code, or pull the template management code into the seeder. That way we can manage the associations between them.

Reply:
done